core-build/gulp-core-build-karma/package.json

```diff
@@ -38,6 +38,7 @@
   "devDependencies": {
     "@microsoft/node-library-build": "~3.0.1",
     "@types/gulp": "3.8.32",
-    "@types/karma": "0.13.33"
+    "@types/karma": "0.13.33",
+    "@types/bluebird": "3.5.3"
```

Comment:
No.  That approach would break anybody who tries to use **gulp-core-build-karma** without re-solving this problem themselves.